packages/blockchain-wallet-v4-frontend/src/data/components/debitCard/sagas.ts

```diff
@@ -49,12 +73,6 @@ export default ({ api, coreSagas, networks }: { api: APIType; coreSagas: any; ne
       try {
         const products = yield call(api.getDCProducts)
         yield put(A.getProductsSuccess(products))
-
-        // If the account is eligible it will get products
-        if (!isEmpty(products)) {
-          // Get the cards the user might have created before
-          yield call(getCards)
-        }
       } catch (e) {
         console.error('Failed to get card products', errorHandler(e))
```

Comment:
again, idk why you have so many `console.error`s here

Reply:
Same answer from above